### DISCUSSION TITLE
是否使用系统字体

### DISCUSSION THREAD

**Discussion by aqni** [Tr0py/NKU-thesis-template-2020#35] [Ideas]
不在模板中内嵌字体，转而使用操作系统自带的字体。

ctex红包可以根据系统的不同自动选择中文字体。

Times New Roman似乎商业操作系统和overleaf都包含。

避免内嵌字体可以规避版权风险。

**Comment by Tr0py** [comment]
如果是的话最好了，但是我记得ubuntu默认不包含Times New Roman 导致docker compile失败，我才加入的。

可以试一下用ctex字体的话，能否在overleaf，docker，ubuntu（或其他linux发行版），macos，windows上正常编译。我们应该最少确保可以在overleaf和docker内可以编译。

**Reply by Tr0py** [reply]
如果你不方便测试各种环境的话（比如docker，ubuntu，macos），可以开一个分支，改用系统字体，我来测试一下。

**Reply by Tr0py** [reply]
避免侵权确实是一个令人头大的问题。
